components/resume/job/dateRange/dateRange.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { FC } from 'react';
+
+import { formatDate } from './dateUtils';
+
+interface DateRangeProps {
+    start: string;
+    end: string | null,
+    fallback?: string
+}
+
+const DateRange: FC<DateRangeProps> = ({ start, end, fallback = 'Present' }) => {
```

Comment:
For now, keeping this as a single component for simplicity of use. There are a few tradeoffs here
- `Present` as the default value is very specific to the current usecase
- the fallback is specific to the end date
- use of `span` and the delimiter `-` is specific to the current usecase

I considered having a `<Date` or `<Formatted` date component that would be more composable, but it would just be a thin wrapper around the `formatDate` function and I'd still need to repeat the `start - end (or Present)` in the two spots where I show dates (per position and at job level).

Going with this route for now to simplify the consumer usage. If I start to notice the usecases diverge, I'll revisit a more composable approach to date formatting.